src/mgr/PyModule.cc

```diff
@@ -362,7 +362,8 @@ int PyModule::load(PyThreadState *pMainThreadState)
       const wchar_t *argv[] = {L"ceph-mgr"};
       PySys_SetArgv(1, (wchar_t**)argv);
       // Configure sys.path to include mgr_module_path
-      string paths = (g_conf().get_val<std::string>("mgr_module_path") + ':' +
+      auto mgr_module_path = g_conf().get_val<std::string>("mgr_module_path");
+      string paths = (mgr_module_path + ":" + mgr_module_path + "/__pypackages__" + ':' +
```

Comment:
If you see the [PEP-582](https://peps.python.org/pep-0582/), the actual path is: `./__pypackages__/lib/python3.10/site-packages/', so you would still need the python version (that usually available as environment variable and/or probably compiler/cmake definition) or use a generic (`python3`).